common-web/shared/src/main/scala/InferWebAsk.scala

```diff
@@ -12,25 +12,15 @@ trait InferWebAsk[Html] {
   implicit def blockCollections[X[_] <: Traversable[_], A]: WebAsk[Html, X[A]] = ???
   implicit def blockCollections2[X[_] <: Traversable[_], A]: WebAsk[Html, X[A]] = ???  
```

Comment:
do these need to be implemented

Reply:
no, they're deliberately there and undefined in order to stop a WebAsk being inductively inferred for a linked list. 
Otherwise it would (for example) render an infinite sequence of radio buttons for cons versus Nil.

By providing two implementations it blocks the implicit resolution by making it ambiguous.